### DISCUSSION TITLE
How to feedback on picamera2-manual.pdf

### DISCUSSION THREAD

**Discussion by cdg123** [raspberrypi/picamera2#225] [General]
I've just come across this great bit of work and read through the draft manual which is also looking great and working through some of the examples for streaming and motion capture.

I noticed a couple of typos and so would like to help by feeding back on those however I am a bit new to the correct area to do this in so should it be in comments here and if so should I cut and paste the section from the pdf or what is best way to help ?

**Comment by sandyol55** [comment]
Probably best to check the 'Closed' Issues list first as quite a few typos have been reported and closed off already.
If you've spotted some not already caught then there is a dedicated Documentation bug report, to be found in the list created by opening a 'New Issue'.
The pro-forma report has instructions.
HTH
